app/controllers/registered_users/boards_controller.rb

```diff
@@ -69,6 +73,14 @@ def get_user
       @user = find_by_username(params[:username])
     end
 
```

Comment:
This may not be the best name for this method, let me know if you think of something better! @akintner @wlffann @CPowell23 @mollybrown